README.md

```diff
@@ -121,6 +121,33 @@ class ContentApp extends Component {
 }
 ```
 
+Also, if you're passing the alias an asynchronous action using something like `redux-thunk` for example, you'll need to have your alias function return a plain javascript object in the interim while the async action is run and include the async middleware after the alias middleware:
+
+```js
+// background.js
+
+import { applyMiddleware, createStore } from 'redux';
+import { alias, wrapStore } from 'react-chrome-redux';
+import thunk from 'redux-thunk';
+
+import { fetchImportantData } from // 'path/to/your/async/action'
+
+const aliases = {
+  'fetch-important-data': (userId) => {
+    // we return an plain javascript object with a type property while the async function runs
+    store.dispatch(fetchImportantData(userId)); // async
```

Comment:
So this is interesting, because this is not how I have been using aliases. Instead of using `store.dispatch()` to call the aliased action, I just return the result of another action creator:

```js
  [ActionTypes.RequestStopCapture]: (action) => {
    return stopCapture(CaptureStopReasons.User);
  },
```

Have you tried this? To your your point and previous points, the alias does need to return an action (or something like a thunk if you're using `redux-thunk`), but I think you can just pass the result of your action creator rather than sending a `NOOP` action.